spec/lib/zipkin_rabbitmq_sender_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'spec_helper'
+require 'zipkin-tracer/zipkin_rabbit_mq_sender'
+
+describe Trace::ZipkinRabbitMqSender do
+  let(:span_id) { ZipkinTracer::TraceGenerator.new.generate_id }
+  let(:trace_id) { Trace::TraceId.new(span_id, nil, span_id, true, Trace::Flags::EMPTY) }
+  let(:rabbit_mq_connection) { double('RabbitMQ connection') }
+  let(:rabbit_mq_exchange) { 'zipkin.exchange' }
+  let(:rabbit_mq_routing_key) { 'routing.key' }
+  let(:publisher) { double('publisher') }
+  let(:tracer) do
+    described_class.new(
+      rabbit_mq_connection: rabbit_mq_connection,
+      rabbit_mq_exchange: rabbit_mq_exchange,
+      rabbit_mq_routing_key: rabbit_mq_routing_key
+    )
+  end
+
+  before do
+    allow(publisher).to receive(:publish)
+    allow(Trace::RabbitMqPublisher).to receive(:new).and_return(publisher)
+  end
+
+  describe '#flush!' do
+    let(:name) { 'test' }
+    let(:span) { tracer.start_span(trace_id, name) }
+    let(:spans) do
+      ::ZipkinTracer::HostnameResolver.new
+        .spans_with_ips([span], described_class::IP_FORMAT)
+        .map(&:to_h)
+    end
+
+    before do
+      Timecop.freeze
```

Comment:
it would be nice for at least one test to show message literal instead of checking variables. this can make clear for example it is a list of json v2 format as opposed to v1

Reply:
Sure, I updated the tests.